Source/Redemption/Private/PauseMenuWidget.cpp

```diff
@@ -0,0 +1,46 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "PauseMenuWidget.h"
+#include "Components/Button.h"
+#include "Kismet/GameplayStatics.h"
+#include "Kismet/KismetSystemLibrary.h"
+
+
+void UPauseMenuWidget::Resume()
+{
+	// Hide the pause menu widget
+	SetVisibility(ESlateVisibility::Hidden);
```

Comment:
every time the player pauses we create a new widget, so I think it is better to call RemoveFromParent() instead of just making it invisible

Reply:
ok